apps/spruce/src/pages/adminSettings/tabs/GeneralTab/OtherTab/schemaFields.ts

```diff
@@ -63,6 +63,24 @@ export const miscSettings = {
         },
       },
     },
+    cost: {
+      type: "object" as const,
+      title: "Cost",
+      properties: {
+        financeFormula: {
+          type: "number" as const,
+          title: "Finance Formula",
+        },
+        savingsPlanDiscount: {
+          type: "number" as const,
+          title: "Savings Plan Discount",
+        },
+        onDemandDiscount: {
+          type: "number" as const,
+          title: "On-Demand Discount",
```

Comment:
nit. for each of these fields, the value must be between 0 and 1. Could we add a description or validation that advises this?